src/components/features/InfoPoper.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react'
+import Box from '@mui/material/Box'
+import Popper, { PopperPlacementType } from '@mui/material/Popper'
+import Grid from '@mui/material/Grid'
+import Button from '@mui/material/Button'
+import Fade from '@mui/material/Fade'
+import Paper from '@mui/material/Paper'
+import InfoIcon from '@mui/icons-material/Info'
+
+interface InfoPopperProps {
+  content: JSX.Element
```

Comment:
use `children` instead of a prop called `content` and when using the component just write this
```tsx
 <InfoPopper>
      <DeliveryGoodsInfo goodTypes={goodTypes} />
</InfoPopper>
```

https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children

Reply:
Fixed